docs/DCAF_101.md

```diff
@@ -0,0 +1,48 @@
+# How Does DCAF Work?
+
+DCAF has many volunteer **case managers** who answer **patient** calls coming in through a hotline phone number and voicemail. DCAF case management software helps the case managers keep track of patient status and needs.
+
+The workflow is similar to many Customer Relationship Manager (CRM) systems. After patients leave a voicemail, the case managers (our users) return their calls and assist patients in navigating the process of scheduling an appointment and raising funds. If the patient is short on money, after a series of phone calls we issue a pledge to support their visit to a clinic. If a patient takes advantage of the pledge and complete their abortion, our accounting team logs that as a **fulfillment**.
+
+[![DCAF High Level User Flow](https://cloud.githubusercontent.com/assets/12372787/11200243/45b1a5de-8ca2-11e5-876c-da0f738041f0.jpg)](https://cloud.githubusercontent.com/assets/12372787/11200243/45b1a5de-8ca2-11e5-876c-da0f738041f0.jpg)
+
+Our case managers work either a one week or a half week shift where they are in charge of one of three lines (DC, VA, or MD/Nationwide). A patient may be touched by several case managers over a period of several weeks, which is why DCAF needed a case management system that more accurately records these interactions across case managers.
+
+## Important concepts, in bullet form
```

Comment:
@lwaldsc mostly I just want to make sure I got these concepts right. Let me know if we need any edits on this!